cloud/migrate-to-cloud/hypershift.md

```diff
@@ -0,0 +1,172 @@
+Your Timescale Cloud trial is completely free for you to use for the first
+thirty days. This gives you enough time to complete all the tutorials and run
+a few test projects of your own.
+</highlight>
+
+## Download the Hypershift container
+
+<Tabs label="Download Hypershift">
+
+<Tab title="MacOS">
+
+<procedure>
+
+### Downloading the Hypershift container on MacOS
+
+1.  Open the Docker app on your local machine to start the service.
+
+1.  At the command prompt, pull the latest Hypershift container from Dockerhub:
+
+    ```bash
+    docker pull timescale/hypershift
+    ```
+
+</procedure>
+
+</Tab>
+
+<Tab title="Linux">
+
+<procedure>
+
+### Downloading the Hypershift container on Linux
+
+1.  Start the Docker service:
+
+    ```bash
+    sudo service docker start
+    ```
+
+1.  At the command prompt, pull the latest Hypershift container from Dockerhub:
+
+    ```bash
+    docker pull timescale/hypershift
+    ```
+
+</procedure>
+
+</Tab>
+
+</Tabs>
+
+## Run Hypershift
+
+In preparation for running Hypershift, you need to determine which tables need
+to be converted to hypertables, and which tables need to be compressed during
+the migration. Hypertables must have a unique column labelled `time`.
+
+Hypershift uses a YAML configuration file to determine how to set up your new
+Timescale database. You can create your own file, or use the example file as a
+starting point. You need these details to complete your configuration file:
+
+*   The schema you want to use
+*   The name of the database you are migrating
+*   The name of the `time` column in your database
+*   The chunk time interval to use
+*   The compression policy you want to use
+
+Use this format:
+
+```yml
+- schema: public
+  name: <DATABASE_NAME>
+  time_column_name: <TIME_COLUMN_NAME>
+  chunk_time_interval: "12h"
+  compress:
+    after: "48h"
+    segmentby:
+      - <COLUMN_NAME>
+    orderby:
+      - time desc
+```
+
+<procedure>
+
+### Running Hypershift
+
+1.  Open the `hypershift.yml` configuration file, and adjust parameters
+    accordingly. For example:
+
+    ```yml
+    - schema: public
+      name: stocks_real_time
+      time_column_name: time
+      chunk_time_interval: "12h"
+      compress:
+        after: "48h"
+        segmentby:
+          - symbol
+        orderby:
+          - time desc
+    ```
+
+1.  At the command prompt, run the Hypershift container. Include the source and
+    destination database connection strings, and the path to your `hypershift.yml`
+    configuration file:
+
+    ```bash
+    docker run -v$(pwd)/hypershift.yml -ti timescale/hypershift:0.2 clone \
+    -s "host=<SOURCE_DB_HOSTNAME> user=postgres port=5431 password=<DB_PASSWORD>" \
+    -t "host=<TARGET_DB_HOSTNAME> user=postgres port=5432 password=<DB_PASSWORD>" \
+    --hypertable /hypershift.yml
+    ```
+
+1.  When the migration has finished, a summary of the migration is shown, and
+    the Docker container exits.
+
+</procedure>
```

Comment:
Do we perhaps want to point people to how they can get help if they have trouble with hypershift? I believe that there will be a channel created in Slack called `#migrations` which will be the avenue by which people should ask for assistance or share their journey.

Reply:
I don't like being too prescriptive on this stuff, because it's likely to change over time. In general, we don't specify where to get help unless it's something specifically that can't be self-serviced (like "Contact support to recover your account" type of thing). I think if you really want to put something in like that we could put in a link to the community page, or to the support form, though.

Reply:
I don't have a strong opinion. My concern is centered

Reply:
I think we can assume that most users would go looking for support, or go through the community Slack/forum. Should be good to go without pointing it out 👍